ddl/src/main/java/com/adaptivescale/rosetta/ddl/DDL.java

```diff
@@ -34,4 +34,16 @@ default String createIndex(Index index) {
     default String dropIndex(Index actual) {
         return null;
     }
+
+    default String dropView(View actual) {
+        return "";
```

Comment:
Is it better to return a **null** or there is a reason for empty string for not supported DBs?

Reply:
returning null causes the DDL to generate lines with string 'null' instead of empty string